IndexOutOfBoundsException occurred when use `ShardingSphere-JDBC`

### Which version of ShardingSphere did you use?

`master`

### Which project did you use? ShardingSphere-JDBC or ShardingSphere-Proxy?

`ShardingSphere-JDBC` and `ShardingSphere-Proxy`

```
Caused by: java.lang.IndexOutOfBoundsException: Index 1 out of bounds for length 1
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64) ~[na:na]
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70) ~[na:na]
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:248) ~[na:na]
	at java.base/java.util.Objects.checkIndex(Objects.java:372) ~[na:na]
	at java.base/java.util.ArrayList.get(ArrayList.java:459) ~[na:na]
	at java.base/java.util.Collections$UnmodifiableList.get(Collections.java:1310) ~[na:na]
	at org.apache.shardingsphere.infra.instance.definition.InstanceDefinition.<init>(InstanceDefinition.java:66) ~[shardingsphere-infra-common-5.1.2-SNAPSHOT.jar:5.1.2-SNAPSHOT]
	at org.apache.shardingsphere.mode.manager.cluster.coordinator.registry.status.compute.service.ComputeNodeStatusService.lambda$null$5(ComputeNodeStatusService.java:163) ~[shardingsphere-cluster-mode-core-5.1.2-SNAPSHOT.jar:5.1.2-SNAPSHOT]
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541) ~[na:na]
	at org.apache.shardingsphere.mode.manager.cluster.coordinator.registry.status.compute.service.ComputeNodeStatusService.lambda$loadAllComputeNodeInstances$6(ComputeNodeStatusService.java:162) ~[shardingsphere-cluster-mode-core-5.1.2-SNAPSHOT.jar:5.1.2-SNAPSHOT]
	at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948) ~[na:na]
	at java.base/java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:658) ~[na:na]
	at org.apache.shardingsphere.mode.manager.cluster.coordinator.registry.status.compute.service.ComputeNodeStatusService.loadAllComputeNodeInstances(ComputeNodeStatusService.java:160) ~[shardingsphere-cluster-mode-core-5.1.2-SNAPSHOT.jar:5.1.2-SNAPSHOT]
	at org.apache.shardingsphere.mode.manager.cluster.ClusterContextManagerBuilder.registerOnline(ClusterContextManagerBuilder.java:168) ~[shardingsphere-cluster-mode-core-5.1.2-SNAPSHOT.jar:5.1.2-SNAPSHOT]
	at org.apache.shardingsphere.mode.manager.cluster.ClusterContextManagerBuilder.build(ClusterContextManagerBuilder.java:75) ~[shardingsphere-cluster-mode-core-5.1.2-SNAPSHOT.jar:5.1.2-SNAPSHOT]
	at org.apache.shardingsphere.driver.jdbc.core.datasource.ShardingSphereDataSource.createContextManager(ShardingSphereDataSource.java:84) ~[shardingsphere-jdbc-core-5.1.2-SNAPSHOT.jar:5.1.2-SNAPSHOT]
	at org.apache.shardingsphere.driver.jdbc.core.datasource.ShardingSphereDataSource.<init>(ShardingSphereDataSource.java:67) ~[shardingsphere-jdbc-core-5.1.2-SNAPSHOT.jar:5.1.2-SNAPSHOT]
	at org.apache.shardingsphere.driver.api.ShardingSphereDataSourceFactory.createDataSource(ShardingSphereDataSourceFactory.java:77) ~[shardingsphere-jdbc-core-5.1.2-SNAPSHOT.jar:5.1.2-SNAPSHOT]
	at org.apache.shardingsphere.spring.boot.ShardingSphereAutoConfiguration.shardingSphereDataSource(ShardingSphereAutoConfiguration.java:93) ~[shardingsphere-jdbc-core-spring-boot-starter-5.1.2-SNAPSHOT.jar:5.1.2-SNAPSHOT]
	at org.apache.shardingsphere.spring.boot.ShardingSphereAutoConfiguration$$EnhancerBySpringCGLIB$$25b7f5e1.CGLIB$shardingSphereDataSource$1(<generated>) ~[shardingsphere-jdbc-core-spring-boot-starter-5.1.2-SNAPSHOT.jar:5.1.2-SNAPSHOT]
	at org.apache.shardingsphere.spring.boot.ShardingSphereAutoConfiguration$$EnhancerBySpringCGLIB$$25b7f5e1$$FastClassBySpringCGLIB$$4410e858.invoke(<generated>) ~[shardingsphere-jdbc-core-spring-boot-starter-5.1.2-SNAPSHOT.jar:5.1.2-SNAPSHOT]
	at org.springframework.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:244) ~[spring-core-5.3.13.jar:5.3.13]
	at org.springframework.context.annotation.ConfigurationClassEnhancer$BeanMethodInterceptor.intercept(ConfigurationClassEnhancer.java:331) ~[spring-context-5.3.13.jar:5.3.13]
	at org.apache.shardingsphere.spring.boot.ShardingSphereAutoConfiguration$$EnhancerBySpringCGLIB$$25b7f5e1.shardingSphereDataSource(<generated>) ~[shardingsphere-jdbc-core-spring-boot-starter-5.1.2-SNAPSHOT.jar:5.1.2-SNAPSHOT]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:na]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
	at java.base/java.lang.reflect.Method.invoke(Method.java:566) ~[na:na]
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:154) ~[spring-beans-5.3.13.jar:5.3.13]
	... 21 common frames omitted
```

--- a/shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-repository/shardingsphere-cluster-mode-repository-provider/shardingsphere-cluster-mode-repository-zookeeper-curator/src/main/java/org/apache/shardingsphere/mode/repository/cluster/zookeeper/listener/SessionConnectionListener.java
+++ b/shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-repository/shardingsphere-cluster-mode-repository-provider/shardingsphere-cluster-mode-repository-zookeeper-curator/src/main/java/org/apache/shardingsphere/mode/repository/cluster/zookeeper/listener/SessionConnectionListener.java
@@ -58,7 +58,7 @@ public final class SessionConnectionListener implements ConnectionStateListener 
         try {
             if (client.getZookeeperClient().blockUntilConnectedOrTimedOut()) {
                 repository.persistEphemeral(ComputeNode.getOnlineInstanceNodePath(instanceContext.getInstance().getCurrentInstanceId(),
-                        instanceContext.getInstance().getInstanceDefinition().getInstanceType()), "");
+                        instanceContext.getInstance().getInstanceDefinition().getInstanceType()), instanceContext.getInstance().getInstanceDefinition().getAttributes());
                 return true;
             }
             sleepInterval();

